packages/nextjs/app/v3/_components/ChooseParameters/EclpParams.tsx

```diff
@@ -80,7 +80,7 @@ function ParamInputs() {
 
   const sortedTokens = tokenConfigs
     .map(token => ({ address: token.address, symbol: token.tokenInfo?.symbol }))
-    .sort((a, b) => (a.address > b.address ? 1 : -1));
+    .sort((a, b) => a.address.localeCompare(b.address));
```

Comment:
I think this poorly executed token sort was the root of issue 4
https://docs.google.com/document/d/1A6C45DXbxb1vRfC6RV7UrH_kzTIRyM2vKdM9iMAo7h8/edit?tab=t.0